README.md

```diff
@@ -1,212 +1,41 @@
-juju
-====
+[![Juju logo](doc/juju-logo.png?raw=true)](https://jujucharms.com/)
 
-juju is devops distilled.
+## Why Juju?
 
-Juju enables you to use [Charms](https://docs.jujucharms.com/applications-and-charms) to deploy your
-application architectures to EC2, OpenStack, Azure, GCE, your data center, and
-even your own Ubuntu based laptop.  Moving between models is simple giving you
-the flexibility to switch hosts whenever you want — for free.
+* Reduce complexity  
+* Enable repeatability  
+* Codify operations knowledge  
+* Simplify day two 
```

Comment:
Suggest 'Simplify day two and beyond' or something similar.  Day two itself is less interesting than days 2-N.

Reply:
+1 that's a really helpful suggestion.

We were thinking about heading away from the leading bullets because it seems to be near-impossible to get the wording right.

Do you think that they're useful?